src/algorithms/tracking/gnuradio_blocks/dll_pll_veml_tracking.cc

```diff
@@ -507,6 +508,7 @@ dll_pll_veml_tracking::dll_pll_veml_tracking(const Dll_Pll_Conf &conf_)
 
     d_multicorrelator_cpu.init(static_cast<int>(2 * d_trk_parameters.vector_length), d_n_correlator_taps);
 
+    d_extend_correlation_symbols = d_trk_parameters.extend_correlation_symbols;
```

Comment:
This could be in the initializer list

Reply:
Thanks. I agree. The initialization may be skipped at all as `d_extend_correlation_symbols` is expected to be always initialized before use in `dll_pll_veml_tracking::start_tracking()`. But static analyzer may still emit warnings...
Wait a minute. I'll move it to initializer list and update a PR.